server/initializers/constants.ts

```diff
@@ -177,6 +177,17 @@ const CONFIG = {
     AUTH: config.has('redis.auth') ? config.get<string>('redis.auth') : null,
     DB: config.has('redis.db') ? config.get<number>('redis.db') : null
   },
+  LDAP: {
+    ENABLE: config.has('ldap.enable') ? config.get<boolean>('ldap.enable') : false,
+    LDAP_ONLY: config.has('ldap.ldap_only') ? config.get<boolean>('ldap.ldap_only') : false,
+    URL: config.has('ldap.url') ? config.get<string>('ldap.url') : null,
+    INSECURE_TLS: config.has('ldap.insecure_tls') ? config.get<boolean>('ldap.insecure_tls') : false,
+    BIND_DN: config.has('ldap.bind_dn') ? config.get<string>('ldap.bind_dn') : null,
+    BIND_PASSWORD: config.has('ldap.bind_password') ? config.get<string>('ldap.bind_password') : null,
+    BASE: config.has('ldap.base') ? config.get<string>('ldap.base') : null,
+    MAIL_ENTRY: config.has('ldap.mail_entry') ? config.get<string>('ldap.mail_entry') : 'mail',
+    USER_FILTER: config.has('ldap.user_filter') ? config.get<string>('ldap.user_filter') : '(|(email=%username%)(uid=%username%))'
```

Comment:
Prefer `null` instead of `'(|(email=%username%)(uid=%username%))'`

Reply:
Ok